frontend/wallet/src/render/views/transactions/Transactions.re

```diff
@@ -192,6 +192,11 @@ let make = () => {
              | Data(data) =>
                let {blocks, pending} = extractTransactions(data);
                let transactions = Array.concatenate(blocks);
+               let lastCursor =
+                 Option.withDefault(
+                   ~default="",
```

Comment:
I thought I remember the cursor being optional instead of "".
I think here it's saying that as long as the string is there it tries to deserialize:
https://github.com/CodaProtocol/coda/blob/1458d2ab6b448dca62fe9ebf8da722d1dd3fcd57/src/lib/coda_graphql/coda_graphql.ml#L858-L864

Did this work against a real daemon?

Reply:
The problem is `graphql_ppx` expects the argument to exist if it's included. This isn't a problem since there will always be a `lastCursor` but not unsure how to do this better.

Reply:
I would have thought it would work to leave it as an option but if that doesn't work and this down then it probably makes sense to leave it.